Error compiling with Typescript v4

```
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/CustomCommandsConfig.d.ts(45,9): error TS2611: 'applicationId' is defined as a property in class 'DialogServiceConfigImpl', but is overridden here in 'CustomCommandsConfig' as an accessor.
```

--- a/src/common/Promise.ts
+++ b/src/common/Promise.ts
@@ -116,8 +116,8 @@ export class Sink<T> {
     private privPromiseResult: PromiseResult<T> = null;
     private privPromiseResultEvents: PromiseResultEventSource<T> = null;
 
-    private privSuccessHandlers: Array<((result: T) => void)> = [];
-    private privErrorHandlers: Array<(e: string) => void> = [];
+    private privSuccessHandlers: ((result: T) => void)[] = [];
+    private privErrorHandlers: ((e: string) => void)[] = [];
 
     public constructor() {
         this.privPromiseResultEvents = new PromiseResultEventSource();

--- a/src/common/Queue.ts
+++ b/src/common/Queue.ts
@@ -117,7 +117,7 @@ export class Queue<TItem> implements IQueue<TItem> {
             }
 
             if (this.privPromiseStore.length() > 0 && pendingItemProcessor) {
-                const promiseArray: Array<Promise<TItem>> = [];
+                const promiseArray: Promise<TItem>[] = [];
 
                 this.privPromiseStore.toArray().forEach((wrapper: Promise<TItem>) => {
                     promiseArray.push(wrapper);

--- a/src/sdk/DialogServiceConfig.ts
+++ b/src/sdk/DialogServiceConfig.ts
@@ -87,10 +87,14 @@ export abstract class DialogServiceConfig {
     public abstract set speechRecognitionLanguage(value: string);
 
     /**
-     * Not used in DialogServiceConfig
-     * @member DialogServiceConfig.applicationId
+     * Sets the corresponding backend application identifier.
+     * @member DialogServiceConfig.prototype.Conversation_ApplicationId
+     * @function
+     * @public
+     * @param {string} value - The application identifier to set.
      */
-    public applicationId: string;
+    // tslint:disable-next-line: no-empty
+    public set applicationId(value: string) { }
 }
 
 /**

--- a/tslint.json
+++ b/tslint.json
@@ -6,6 +6,8 @@
   "extends": "tslint:recommended",
   "rules": {
 
+    "array-type": [ true, "array" ],
+
     "interface-name": [ false ],
 
     "max-line-length": [ false ],

